pallets/msa/src/tests.rs

```diff
@@ -1533,7 +1506,7 @@ fn register_provider_max_size_exceeded() {
 		let (_new_msa_id, _) =
```

Comment:
I think this test should be renamed to create_provider_max_size_exceeded, but unless you are going to change something else, I would not hold up the PR.